### ISSUE TITLE
ClassiK Application Style breaks background transparency on Yakuake

### ISSUE THREAD

**Issue by rocka** [paulmcauley/classik#37]
Yakuake: Drop-down terminal emulator based on Konsole technologies
https://invent.kde.org/utilities/yakuake

**System info:**

KDE Plasma Version: 5.25.0
KDE Frameworks Version: 5.95.0
Qt Version: 5.15.5
Graphics Platform: X11

ClassiK Version: d83c45f622deb42b5e1c1798c1dee85392f8aec0 (AUR package [classik-git](https://aur.archlinux.org/packages/classik-git))

**Step to reproduce:**

- select ClassiK in system settings -> Appearance -> Application Style
- restart Yakuake

**Observed behavior:**

No background transparency, shaodw become solid as well

![image](https://user-images.githubusercontent.com/13914967/174747320-cdfc85d0-59b1-4881-931c-a4f90aa2c6dc.png)

**Expected behavior:**

Semi-transparent background and shadow (screenshot took with Breeze Application Style)

![image](https://user-images.githubusercontent.com/13914967/174747444-9d09d22a-c209-4397-9b24-69cfa898b313.png)

**Additional info:**

Yakuake theme used in screenshot: https://github.com/noahadvs/yakuake-breeze_perfect_dark

**Comment by paulmcauley** [comment]
I can't reproduce this, even with Breeze. Here is what Yakuake looks like for me with Breeze:
![yakuake](https://user-images.githubusercontent.com/6924479/174791585-a582f810-a014-4a78-ab5f-85f8f788be7a.png)

I see no translucency or shadow with Breeze. Have you made more customizations?

When I switch application style to ClassiK, Yakuake looks identical.

**Comment by rocka** [comment]
I have 20% background transparency in my Konsole profile, which applies to yakuake as well:

![image](https://user-images.githubusercontent.com/13914967/174819224-2a9e758c-c634-4796-a4eb-1952fa3ecdb5.png)

I noticed that it says "Konsole was started before desktop effects were enabled" in Yakuake's right click -> "Edit Current Profile" dialog, but it's not the case, desktop effects were always enabled.

But with Breeze Application Style, there is no such warning.

**Comment by paulmcauley** [comment]
If you enable transparency in the titlebar tab of the ClassiK window decoration it controls whether or not the window can have transparency. I will look into the logic of this and see if it can prevent existing windows from having transparency inhibited...

I can now get translucency in the Yakuake window, but how did you get a shadow?

**Comment by rocka** [comment]
I set titlebar transparency to 99% in ClassiK window decoration and it worked.

As for shadow, I looked into the yakuake theme I mentioned, the shadow varient was removed in https://github.com/noahadvs/yakuake-breeze_perfect_dark/commit/be3042bd28e711fd8d081745c1a074e820f38ee2 , and I'm using an old version (yes, since 2018 ...).

The actual commit I'm using is https://github.com/noahadvs/yakuake-breeze_perfect_dark/tree/f88374884fc41e47361338af29dfb351d9d4ad2b , which enabled shadow by default.
